www/site/content/docs/chef-workstation/getting-started.md

```diff
@@ -12,7 +12,7 @@ Chef Workstation gives you everything you need to get started with Chef. Ad-hoc
 
 ## Install Chef Workstation
 
-If you have not installed Chef Workstation, download and install it from the [chef.sh](https://www.chef.sh) website.
+If you have not installed Chef Workstation, download and install it from the [chef.sh](/) website.
```

Comment:
This takes you to https://www.chef.sh/ but then users still need to click `Downloads` which takes them to downloads.chef.io - why not just go there?

Reply:
yeah i can update it. I was trying not to change the functionality.

Reply:
it should take you to localhost so you've got some caching. The only reason I can think of that they wanted to go to sh is the the button make sure they get the latest.

Reply:
Actually thinking about this it's the same number of steps because the down load button figures out version and platform for the user.